### PR TITLE
Decaf proxy tests

### PR DESCRIPTION
**Pull request by flotwig**
:-1: |no_squash
-|-
:+1:|image

### CODE REVIEW

File: packages/server/test/unit/blacklist_spec.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// TODO: This file was created by bulk-decaffeinate.
```
Comment (Bkucera) [inline packages/server/test/unit/blacklist_spec.js]: remove comment

File: packages/server/test/unit/cors_spec.js [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+
+    it('parses http://www.local.nl', function () {
+      return this.isEq('http://www.local.nl', {
+        port: '80',
+        domain: 'local',
+        tld: 'nl',
+      })
+    })
+
+    //# https://github.com/cypress-io/cypress/issues/3717
```
Comment (Bkucera) [inline packages/server/test/unit/cors_spec.js]: find-replace all `//#` with `//`

File: packages/server/test/unit/security_spec.js [hunk 3]
```diff
@@ -0,0 +1,309 @@
+/* eslint-disable
```
Comment (Bkucera) [inline packages/server/test/unit/security_spec.js]: remove + fix eslint errors

Comment (flotwig) [reply packages/server/test/unit/security_spec.js]: would be cool if eslint was hooked up to do this automatically

**Pull request merged**